data/en/mbs_0167.json

```diff
@@ -0,0 +1,662 @@
+{
+    "title": "Monthly Business Survey",
+    "survey_id": "009",
+    "mime_type": "application/json/ons/eq",
+    "theme": "default",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "mbs",
```

Comment:
eq_id and form types needed to be added for this too

Reply:
See my other comment